network/init-neutrond.sh

```diff
@@ -719,6 +719,8 @@ set_genesis_param proposer_fee                          "\"0.25\""
 set_genesis_param escrow_account_address                "\"$DAO_CONTRACT_ADDRESS_B64\","                  # builder(POB)
 set_genesis_param sudo_call_gas_limit                   "\"1000000\""                                     # contractmanager
 set_genesis_param max_gas                               "\"1000000000\""                                  # consensus_params
+set_genesis_param vote_extensions_enable_height         "\"1\""                                           # consensus_params
+set_genesis_param market_authority                      "\"neutron1hxskfdxpp5hqgtjj6am6nkjefhfzj359x0ar3z\","                      # marketmap
```

Comment:
what is this address? why not a const?

Reply:
this is admin module address, value used for the check of who can change data inside marketmap module.
not sure if constant makes sense here.